### PR TITLE
lesson10 Task5

### PR DESCRIPTION
**Pull request by TeresaTao127**

### CODE REVIEW

File: src/main/java/lesson10Task5/PasswordValidator.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package lesson10Task5;
+
+
+import java.util.regex.*;
+
+public class PasswordValidator {
+    public static void main(String[] args) {
```
Comment (strzeleckik) [inline src/main/java/lesson10Task5/PasswordValidator.java]: you should create some unit test for that

**Review by strzeleckik — CHANGES_REQUESTED** [review]
add some unit tests for positive and negative scenario
